README.md

```diff
@@ -1000,6 +1000,10 @@ If you run into an issue with this tool or would like to request a feature you c
 
 If you have problems getting this tool up and running, please contact Support.
 
+## License
```

Comment:
Can you please add it to the table of contents as well?

Reply:
Yep! On it now - thank you for catching this :-)